tests/functional/adapter/hooks/test_on_run_hooks.py

```diff
@@ -125,7 +125,7 @@ def test_results(self, project):
         ]
 
         assert [(result.node.unique_id, result.status) for result in results] == expected_results
-        assert "PASS=3 WARN=0 ERROR=1 SKIP=1 TOTAL=5" in log_output
+        assert "PASS=3 WARN=0 ERROR=1 SKIP=1 NO-OP=0 TOTAL=5" in log_output
```

Comment:
because of test duplication, we'll need to update this in dbt-tests-adapters as well + cut a release to avoid breaking downstream concrete adapter tests: https://github.com/dbt-labs/dbt-adapters/blob/main/dbt-tests-adapter/dbt/tests/adapter/concurrency/test_concurrency.py#L329.

I'd opt for removing this assertion from the dbt-adapters tests. It seems brittle to test against core logs.